### PR TITLE
[#2099] fix(web): redirect to login page if token has expired

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

Remove refresh token and redirect to login page if token has expired.

### Why are the changes needed?

Fix: #2099 

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by jerryshao** [comment]
There's another solution. if the token is expired and returns 401, we can redirect to the login page to ask users to re-login. WDYT @qqqttt123 ?

**Comment by qqqttt123** [comment]
> There's another solution. if the token is expired and returns 401, we can redirect to the login page to ask users to re-login. WDYT @qqqttt123 ?

This may affect the experience of users.

**Comment by jerryshao** [comment]
> > There's another solution. if the token is expired and returns 401, we can redirect to the login page to ask users to re-login. WDYT @qqqttt123 ?
> 
> This may affect the experience of users.

Yeah, but it is a common way as far as I know. We cannot renew the token forever, it is not safe.

**Comment by jerryshao** [comment]
@ch3yne Can you please update the codes to the logic we mentioned above?

**Comment by ch3yne** [comment]
> @ch3yne您能否将代码更新为我们上面提到的逻辑？

Done.

**Comment by xunliu** [comment]
@LauraXia123 Please help review this PR, thanks!

File: web/lib/utils/axios/checkStatus.ts [hunk 1]
```diff
@@ -39,12 +38,13 @@ export function checkStatus(status: number, msg: string, errorMessageMode: Error
...
     case 401:
+      // ** reserve error message
       errMessage = msg || 'The user does not have permission (token, user name, password error or expired)!'
 
       localStorage.removeItem('accessToken')
-      localStorage.removeItem('version')
+      localStorage.removeItem('authParams')
 
-      Router().push('/ui/login')
+      window.location.href = '/ui/login'
```
Comment (LauraXia123) [inline web/lib/utils/axios/checkStatus.ts:47]: image
For Server Components, use the redirect function instead.

**Review by LauraXia123 — APPROVED** [review]

**Comment by jerryshao** [comment]
@ch3yne would you please update the PR title and description? The current change is not about refreshing the token any more.

**Comment by ch3yne** [comment]
> @ch3yne would you please update the PR title and description? The current change is not about refreshing the token any more.

Done

**Pull request merged**